apps/qubit/modules/term/actions/deleteAction.class.php

```diff
@@ -45,13 +45,7 @@ public function execute($request)
 
     if ($request->isMethod('delete'))
     {
-      foreach ($this->resource->descendants->andSelf()->orderBy('rgt') as $item)
-      {
-        if (QubitAcl::check($item, 'delete'))
```

Comment:
Is this ACL check still necessary? It looks like it has been removed.

Reply:
Good point @sbreker! Sorry, I forgot to add a note about this ...

The ACL permissions for terms are set at the taxonomy level so, if the user has permission to delete a term, it will also have it to delete its descendants. But, now that you say it, we didn't have that check for IOs either, where the permissions can be set at the IO level. However, I'm not sure if we want to add it as it would be a rare case and it will seriously affect the performance of this hierarchy deletions.

At least this is something @fiver-watson should know, maybe we want to add a note in the documentation if we don't have it already or address it in a different issue.